### PR TITLE
Add icon button pattern, new icons DAH-157

### PR DESCRIPTION
**Pull request by akegan**
This PR:
1. Makes the status column wider for the lease-up application table
1. Adds the button .icon styling 
1. Adds the new list2 and equalizer icons for the status history popover and upcoming filter work
1. Re-downloaded all of the current icons and added instructions for adding new icons in the future
image

To review:
1. Pull up the pattern library locally
1. Look at the new icon buttons and see if they look reasonable: http://localhost:3001/components/detail/icon-button
1. Review old icons and make sure that they look ok, and that new equalizer and list2 bulleted list icons are showing up: http://localhost:3001/components/detail/icon-all

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ To deploy the application:
     * For example: `./deploy.sh testing` will deploy to `sf-dahlia-pl-testing.herokuapp.com`
   * This works from any branch, for example if you have checked out the `dev` branch and you `./deploy.sh new-menu` then the dev branch will be deployed to `sf-dahlia-pl-new-menu.herokuapp.com`
 
+## Adding New Icons
```
Comment (james-wills-sf) [inline README.md:32]: Thank you for this!

Just checking, if we ever have a custom icon from Chris, I'm guessing the process will be totally different, right? I guess we can just keep in mind to update this documentation if that ever happens

File: public/toolkit/icons/icomoon/demo.html [hunk 2]
```diff
@@ -8,231 +8,194 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (james-wills-sf) [inline public/toolkit/icons/icomoon/demo.html:11]: Could this file be added to gitignore?

**Review by james-wills-sf — APPROVED** [review]
@akegan lgtm, following are just ideas you can take or leave, I don't need to re-review.

Is there something semantically different about list2 we could use for the name instead of just adding a number to it?

Maybe `i-bullet-list` or `i-list-unordered` since the other list is ordered/numbered?

Also just generally maybe we can add some of the icomoon demo files to our .gitignore to reduce the noise of the diff

File: README.md (hunk 1 above)
Comment (akegan) [reply README.md:32]: yeah we'll have to do that for the numbered circles, add them outside of the icomoon folder definitely.

File: public/toolkit/icons/icomoon/demo.html (hunk 2 above)
Comment (akegan) [reply public/toolkit/icons/icomoon/demo.html:11]: hm probably as long as it's still in there for people to copy over

Comment (james-wills-sf) [reply public/toolkit/icons/icomoon/demo.html:11]: @akegan ah is this file used for anything? I was under the impression it doesn't do anything and is just for reference

If it's being used then it should probably be kept up to date.

**Comment by akegan** [comment]
does turn out that we can rename the icons in icomoon and then keep those renames so I went ahead and re-named to list-unordered. I also added a couple of demo files to gitignore

File: public/toolkit/icons/icomoon/demo.html (hunk 2 above)
Comment (akegan) [reply public/toolkit/icons/icomoon/demo.html:11]: when we generate a new icomoon download we have to use demo.html to copy and paste values into _icons.html but I think you're right that we don't need it to be checked in

**Pull request merged**